docs/shone/examples/chemistry.rst

```diff
@@ -0,0 +1,309 @@
+    names = species_table.loc[species]['name']
+
+    ax = plt.gca()
+    ax.loglog(vmr[:, indices], pressure, label=names)
+    ax.legend(loc='lower left')
+    ax.invert_yaxis()
+    ax.set(
+        xlabel='Volume mixing ratio',
+        ylabel='Pressure [bar]'
+    )
+
+.. plot::
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from shone.chemistry import FastchemWrapper
+
+    pressure = np.geomspace(1e-6, 10, 15)  # [bar]
+    temperature = 2300 * (pressure / 0.1) ** 0.1  # [K]
+
+    chem = FastchemWrapper(
+        temperature, pressure,
+        # these are in linear (not log) units:
+        metallicity=1,  # M/H
+        c_to_o_ratio=1  # C/O
+    )
+    species_table = chem.get_species()
+    vmr = chem.vmr()
+
+    species = ['H2O1', 'O1Ti1', 'e-']
+    indices = species_table.loc[species]['index']
+    names = species_table.loc[species]['name']
+
+    ax = plt.gca()
+    ax.loglog(vmr[:, indices], pressure, label=names)
+    ax.legend(loc='lower left')
+    ax.invert_yaxis()
+    ax.set(
+        xlabel='Volume mixing ratio',
+        ylabel='Pressure [bar]'
+    )
+
+Mass mixing ratio
++++++++++++++++++
+
+The mass mixing ratio (MMR) is equivalent to the volume mixing ratio multiplied
+by the mass of the species and divided by the mean molecular weight. Since
+mean molecular weight is often a free parameter in a real atmospheric retrieval, we
+can't return "one MMR" per FastChem run. So `~shone.chemistry.FastchemWrapper` has
+a method called `~shone.chemistry.FastchemWrapper.mmr_mmw` which returns the VMR
+multiplied by the molecular mass, which is equivalent to the MMR multiplied by
+the mean molecular weight. To convert this to mass mixing ratio, simply divide
+the result of `~shone.chemistry.FastchemWrapper.mmr_mmw` by the mean molecular weight.
+
+Opacity grids are often stored as cross sections per mass, usually written
+as :math:`\kappa` [:math:`{\rm cm}^2~{\rm g}^{-1}`]. To
+compute an extinction coefficient :math:`\alpha` [:math:`{\rm cm}^{-1}`],
+we must multiply the opacity :math:`\kappa` by the mass density of the species :math:`\rho`
+[:math:`{\rm g~cm}^{-3}`].
+
+
+Precompute FastChem on grid
+---------------------------
+
+Build a grid
+++++++++++++
+
+As the name suggests, the FastChem is fast! That said, computing it millions of times
+during Monte Carlo sampling may not be the best use of your time for species with mixing
+ratios that vary smoothly with temperature, pressure, M/H, and C/O. We have included a
+convenience function called `~shone.chemistry.build_fastchem_grid` that runs FastChem in
+a loop over these four dimensions to create a ~100 MB grid of abundances for each species
+in less than a minute on a laptop:
+
+.. code-block:: python
+
+    from shone.chemistry import build_fastchem_grid
+
+    chem_grid = build_fastchem_grid()
+
+The grid is saved to your `~/.shone` directory and can be interpolated during sampling to
+use *approximate* FastChem mixing ratios.
+
```

Comment:
State here what the defaults are? Temperature between 300 to 6000 in 22 steps, pressure logarithmically from 1e-8 to 10 bars in 20 steps, metallicity from -1 to 2.5 in 8 steps; log(c/o) from -1 to 2. 

Is that, by the way, a sensible range? When we say solar C/O is 0.52; that's not log(C/O), right?